data/mods/Backrooms/items.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "id": "jug_cooler",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "water cooler jug" },
+    "looks_like": "bottle_plastic",
+    "description": "A sizable blue plastic jug meant to hold and supply water in a water cooler or similar dispenser.  Holds a lot of liquid, but unfortunately it doesn't come with a cap, making it hard to carry around when full.",
```

Comment:
I see mainline content viability 👀

Reply:
I'm gonna resolve this Convo but I agree water coolers could be moved if there was desire.